### ISSUE TITLE
cypress opens off screen

### ISSUE THREAD

**Issue by burtonrodman** [cypress-io/cypress#4606]
### Current behavior:
my desktop configuration recently changed.
ran npm run cypress open
the cypress icon appeared in the taskbar, but the window was nowhere to be seen.
hit Alt+Space, M and moved the window back onto the desktop view.

### Desired behavior:
when opening the cypress window, check that it is within the currently defined desktop coordinates.

### Versions
cypress 3.3.1
Windows 10 1809.

**Comment by jennifer-shehane** [comment]
>my desktop configuration recently changed.

Could you describe exactly what changed here to make Cypress behave in this way? 

My guess is we had your placement/sizing configuration saved before in your app's App Data and after changing, it's still trying to open up as before. We think opening from the previous sizing and placement is better than trying to recalculate each time and wipe any way you had it set up on your desktop before.

Can you clear your app data and then reopen Cypress? Does it open correctly after that? https://on.cypress.io/debugging#Clear-App-Data

**Comment by burtonrodman** [comment]
Yes, after repositioning the window once, it was saved and everything was fine.  My point is that opening completely outside the bounds of the _current_ desktop dimensions is a poor user experience.

i have a Surface Book 2 with 2 external 1920p monitors.  I arrange my Desktop such that the Surface is centered below the 2 external monitors which are side-by-side.  I don't recall exactly what happened, but I had to reset the position of the external monitors and restore the dpi of the Surface Book display.

Unfortunately, since I work in so many different scenarios (home office, shared office space, conference rooms, etc...) this is a fairly common occurrence.

I agree that saving the last position of the cypress window is the right way to keep it.  However, opening in a coordinate that is completely invisible is a bad experience.  I'm simply suggesting that this is easily avoided by adding a check of the saved coordinate/size against the _current_ desktop geometry.

**Comment by franciscoD96** [comment]
so I cannot manage to grab the window, because I worked with an external monitor which I don't have right now and I can't pick it up.

**Comment by pbking** [comment]
Same happens to me.  It's only inconvenient... once you are reminded of the alt+space, M trick to get it back.  It's downright impossible to use otherwise.

**Comment by jennifer-shehane** [comment]
Clearing app data should reset the location of the Cypress application to default. Follow [these instructions](https://on.cypress.io/troubleshooting#Clear-App-Data) to Clear App Data.

**Comment by Bjodol** [comment]
To those who had a problem with this: My default location was `/home/<USERNAME>/.config/Cypress/cy/`. 
I found it by running:
```sh
find / | grep /cy/
```
Run `rm -rf <LOCATION>/cy/*` to reset the viewport

**Comment by ronaldheg** [comment]
What I did was to select the cypress window. Once is selected click on alt+space, then a floating options will appear so click on maximise. Finally, resize the window. :)

**Comment by johnknoop** [comment]
In case one forgets about @ronaldheg nice workaround, it would be good with "move Cypress to current display" option when right-clicking the icon on the task bar, just like Slack offers:

![image](https://user-images.githubusercontent.com/3394270/185074299-2944c71c-53e0-43b6-b8cf-c84f19c5b1db.png)

**Comment by EmilyThomasPCTY** [comment]
This is inconvenient and happens to me every few months when I have the  need to work away from my desk. I hope this is eventually prioritized.
